src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -67,6 +79,16 @@ public SscsCaseData getOldSscsCaseData() {
         return responseWrapper.getOldSscsCaseData();
     }
 
+    @Override
+    public List<SubscriptionWithType> getSubscriptionsBasedOnNotificationType() {
+        List<SubscriptionWithType> subscriptionWithTypeList = new ArrayList<>();
+        subscriptionWithTypeList.add(new SubscriptionWithType(getAppellantSubscription(), APPELLANT));
+        if (SYA_APPEAL_CREATED_NOTIFICATION.getId().equals(getNotificationType().getId())) {
```

Comment:
Not sure why the representative subscription is created based on Id. It should be based on whether the representative subscription exists and is subscribed for email or sms notification

Reply:
Yes, both subs are validated to confirm they are not null and there is some subscribed present either sms or email. But that is not done here. This method is just a wrapper to add the subscription Type: appellant or reps. Which needs to be passed down in the next methods.